src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1096,7 +1096,7 @@
     <value>Partial declarations of '{0}' have conflicting accessibility modifiers</value>
   </data>
   <data name="ERR_PartialMultipleBases" xml:space="preserve">
-    <value>Partial declarations of '{0}' must not specify different base classes</value>
+    <value>Partial declarations of '{0}' must not specify different base types</value>
```

Comment:
>types [](start = 73, length = 5)

I think it is fine to specify different interfaces though.

Reply:
I thought interfaces didn't count in "base type", but looks like I was wrong ([spec](https://github.com/dotnet/csharplang/blob/master/spec/classes.md#type-parameter-constraints)).

I'll revert to "base classes" here then, since records are classes.